dev/README.md

```diff
@@ -116,17 +116,185 @@ incompatible way at any day.
 
 ## Requirements
 
-* docker
-* git
-* go
-* helm
-* jq
-* k3d
-* kubectl
-* ...
+- docker
+- git
+- go
+- helm
+- jq
+- k3d
+- kubectl
+- ...
 
 ## Run integration tests
 
     ./dev/run-integration-tests.sh
 
 This will download and prepare setup-envtest, then it will execute all the integration tests.
+
+## Running Github Actions locally
+
+Sometimes, it may be beneficial to be able to run the Github Action tests using
+the same configuration which is used remotely. To do this, you can use
+[nektos/act](https://github.com/nektos/act).
+
+### Requirements
+
+- [Docker](https://www.docker.com/)
+- [act](https://github.com/nektos/act#installation)
+
+### Installation
+
+To install `act`, please follow the [instructions on the official Github
+repository](https://github.com/nektos/act#installation).
+
+### Configuration
+
+#### Container Image
+
+Unlike Github Actions, `act` will use container images for running the tests
+locally. The container image that will be used depends on the type of the
+Github Action Runner for the specific action. You can see which images are being
+used for which runner [here](https://github.com/nektos/act#runners). Most tests require `ubuntu-latest`.
+
+The containers are available in difference sizes:
+
+- Micro
+- Medium
+- Large
+
+The default container image for Ubuntu, which is required for most actions, does
+not contain all necessary tools. They are [intentionally
+incomplete](https://github.com/nektos/act#default-runners-are-intentionally-incomplete).
+Instead, the large container image needs to be used (read on for an
+alternative).
+
+To change the container image used for Ubuntu, you will need to create a
+configuration file `$HOME/.actrc`.
+
+```shell
+-P ubuntu-latest=catthehacker/ubuntu:full-latest
```

Comment:
When I run `act` on a Mac it also suggest adding `--container-architecture linux/amd64`. I think it's a sane default since our CI assumes `linux/amd64` arch is used (I'm going to try it without it, though)

Reply:
The `--pull=false` flag seems to work well. I'll document that and drop the pre-built container image from a non-rancher namespaces.